cpp/include/gpu_operations.h

```diff
@@ -47,6 +56,54 @@ class GpuOperations {
   static Matrix<T> OuterProduct(const Vector<T> &a, const Vector<T> &b);
 };
 
+template <typename T>
+Matrix<T> GpuOperations<T>::Multiply(const Matrix<T> &a, const Matrix<T> &b) {
+  if (a.cols() == b.rows()) {
+  int m = a.rows();
```

Comment:
If this is a new file just added to include directory, we need to rerun cmake .. in order to use cpplint.py on this file right?